src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -145,7 +145,7 @@ static TDSType valueOf(int intValue) throws IllegalArgumentException {
     DECIMAL        (Category.NUMERIC,         "decimal",          JDBCType.DECIMAL),
     NUMERIC        (Category.NUMERIC,         "numeric",          JDBCType.NUMERIC),
     GUID           (Category.GUID,            "uniqueidentifier", JDBCType.GUID),
-    SQL_VARIANT    (Category.VARIANT,         "sql_variant",      JDBCType.VARCHAR),
+    SQL_VARIANT    (Category.SQL_VARIANT,     "sql_variant",      JDBCType.VARCHAR),  
```

Comment:
I may be very picky here. Should we remove `_` underscore here and just call them SQLVARIANT and sqlvariant? since no other types use underscore. Just my idea, I am also good to leave it as is.